File: v2/jaxrs/src/main/java/io/swagger/model/Newsletterresponse.java
package io.swagger.model;

import io.swagger.model.Post;

import com.wordnik.swagger.annotations.*;
import com.fasterxml.jackson.annotation.JsonProperty;


@ApiModel(description = "")
public class Newsletterresponse  {
  
  private Post newsletter = null;

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("newsletter")
  public Post getNewsletter() {
    return newsletter;
  }
  public void setNewsletter(Post newsletter) {
    this.newsletter = newsletter;
  }

  

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class Newsletterresponse {\n");
    
    sb.append("  newsletter: ").append(newsletter).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

File: v2/csharp/src/main/csharp/io/swagger/Model/Newsletterresponse.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace io.swagger.Model {
  public class Newsletterresponse {
    

    
    public Post Newsletter { get; set; }

    

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class Newsletterresponse {\n");
      
      sb.Append("  Newsletter: ").Append(Newsletter).Append("\n");
      
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  
  
}